package.json

```diff
@@ -24,8 +24,11 @@
     "@atom-learning/components": "../@atom-learning-components",
     "@atom-learning/theme": "^0.1.0-beta.5",
     "@next/mdx": "^10.0.6",
-    "@stitches/react": "^0.1.0-canary.9",
+    "@stitches/react": "0.1.0-canary.5",
```

Comment:
What's the reason for reverting here?

Reply:
I was getting some weird TS errors with a later beta version of stitches, I tried to fix them as part of this PR but I didn't think it was necessary to include those updates as part of this PR so downgraded for stability and consistency with the components library.

Reply:
Cool, seems sensible! The perils of early adoption ...